src/plugins/intel_cpu/src/exec_network.cpp

```diff
@@ -100,6 +100,12 @@ ExecNetwork::ExecNetwork(const InferenceEngine::CNNNetwork &network,
     } else {
         auto streamsExecutorConfig = InferenceEngine::IStreamsExecutor::Config::MakeDefaultMultiThreaded(_cfg.streamExecutorConfig, isFloatModel);
         streamsExecutorConfig._name = "CPUStreamsExecutor";
+        _cfg.streamExecutorConfig._threads =
```

Comment:
Just wondering why threads number is updated not inside "MakeDefaultMultiThreaded"

Reply:
Because streamsExecutorConfig returned from "MakeDefaultMultiThreaded" is just a temporary variable, it is not updated in  _cfg.streamExecutorConfig. But we can place calculating in "MakeDefaultMultiThreaded", place assignment here.